File: sdkwork-app-sdk-typescript/src/types/vip-renew-form.ts
/** VIP续费请求 */
export interface VipRenewForm {
  /** 续费天数 */
  duration?: number;
  /** 套餐ID */
  packId?: number;
  /** 优惠券ID */
  couponId?: string;
  /** 支付方式 */
  paymentMethod?: string;
}

File: sdkwork-app-sdk-csharp/Models/VipRenewForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VipRenewForm
    {
        public int? Duration { get; set; }
        public int? PackId { get; set; }
        public string? CouponId { get; set; }
        public string? PaymentMethod { get; set; }
    }
}
